src/guides/v2.3/graphql/custom-filters.md

```diff
@@ -7,7 +7,7 @@ As of Magento 2.3.4, the `filter` attribute of the [`products`]({{page.baseurl}}
 
 ## Prerequisites
 
-You have several options when enabling a custom attribute (or any attribute that is not listed by default in the `ProductAttributeFilterInput` object) for filtering. Navigate to the attribute's **Storefront Properties** page (**Stores** > Attributes > **Product** > <attribute name> > **Storefront Properties**) in the Admin, then perform one or both of the following actions:
+You have several options when enabling a custom attribute (or any attribute that is not listed by default in the `ProductAttributeFilterInput` object) for filtering. Navigate to the attribute's **Storefront Properties** page (**Stores** > Attributes > **Product** > attribute name > **Storefront Properties**) in the Admin, then perform one or both of the following actions:
```

Comment:
I feel this is still not clear enough as the instruction implies that it's a single menu chain that needs to be followed, which is not what is needed. I would recommend something like -

Navigate to the attribute's **Storefront Properties** section -
1. Select **Stores** > Attributes > **Product**
2. Click on required attribute in the grid
3. Click on **Storefront Properties** tab

Reply:
@denish-c-vachhani  I'm going to override @BarnyShergold's suggestion. The path is fine as is. However, it stead of deleting the less than/greater than signs, replace them with their HTML equivalents `&lt;` and `&gt;`